source/sass/buyers-guide/views/category.scss

```diff
@@ -0,0 +1,89 @@
+.category-listing {
```

Comment:
Most of this page layout should be achievable with Bootstrap and other existing type classes (see https://foundation.mozilla.org/en/style-guide/), but I'll leave it up to you if you want to refactor given time constraints.

Reply:
it's  good idea, although I will need to learn myself some bootstrap before I can do this, so I'll file it as a followup: https://github.com/mozilla/foundation.mozilla.org/issues/1924